From d18485ccb549f6ea7bc54560b299665ce08cec82 Mon Sep 17 00:00:00 2001
From: Kevin Lin <kevinptt@google.com>
Date: Wed, 4 Mar 2026 23:39:46 +0800
Subject: [PATCH] Add [[maybe_unused]] to wrapped_ptr in
 RawPtrNoOpImpl::ReleaseWrappedPtr.

This silences compiler warnings about the unused parameter in the no-op implementation, especially when the MSAN check is skipped.

Bug: None
Test: CQ
Change-Id: I146d5ad5e99f26c3ad5dfa36034d7bf5279f9e41
---
 .../src/partition_alloc/pointers/raw_ptr_noop_impl.h           | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/base/allocator/partition_allocator/src/partition_alloc/pointers/raw_ptr_noop_impl.h b/base/allocator/partition_allocator/src/partition_alloc/pointers/raw_ptr_noop_impl.h
index e4bb2ca617..772a43b983 100644
--- a/base/allocator/partition_allocator/src/partition_alloc/pointers/raw_ptr_noop_impl.h
+++ b/base/allocator/partition_allocator/src/partition_alloc/pointers/raw_ptr_noop_impl.h
@@ -26,7 +26,8 @@ struct RawPtrNoOpImpl {
   // Notifies the allocator when a wrapped pointer is being removed or
   // replaced.
   template <typename T>
-  PA_ALWAYS_INLINE static constexpr void ReleaseWrappedPtr(T* wrapped_ptr) {
+  PA_ALWAYS_INLINE static constexpr void ReleaseWrappedPtr(
+      [[maybe_unused]] T* wrapped_ptr) {
     // Other RawPtrImpls use wrapped_ptr here. Check that it's initialized.
     if (!std::is_constant_evaluated()) {
       PA_MSAN_CHECK_MEM_IS_INITIALIZED(&wrapped_ptr, sizeof(wrapped_ptr));
-- 
2.53.0.473.g4a7958ca14-goog

